code_gen/README.md

```diff
@@ -0,0 +1,34 @@
+# AI Code Gen for Pipedream Components
```

Comment:
I'd recommend moving all of this into the `packages` directory, and maybe rename to `component_code_gen` just to clarify (since we may end up doing other code gen in the future).